### ISSUE TITLE
A11y: PanelChrome fixed header and children are accessible through a keyboard and screen reader

### ISSUE THREAD

**Issue by polibb** [grafana/grafana#59082]
Part of https://github.com/grafana/grafana/issues/56163.
Discuss scope after the other tasks are done.
- [ ] Allow focusing on the container of a `PanelStateWrapper` => moving from a panel to a panel with Tab.
- [ ] Focusing on a panel allows for a focus on each item in the panel separately by going inside the panel contents with keyboard arrows.
  - [ ] State of the panel (Error, Warning, Data Info) is the first one of the interactions of each panel to be focused, if it exists.  
- [ ] User can open a menu and interact with it with keyboard arrows.

No >=2 interactions are on top of one another.

The focus area and sequence of selection should be based [on the designs](https://www.figma.com/file/jdTCXlq75p55itRRSaerFx/Panel-header-and-menu?node-id=2279%3A3&t=YKsglklbrVYIFU92-1).

**Comment by GunFood** [comment]
WCAG is still important!
